firmware/fsm.c

```diff
@@ -56,6 +56,8 @@
 #include "rfc6979.h"
 #include "gettext.h"
 
+_Static_assert(pb_arraysize(Features, coins) >= COINS_COUNT, "Features.coins max_count not large enough");
```

Comment:
Sorry to be irritating about this, but I just realised you could put the `_Static_assert` inside `fsm_msgGetFeatures` which might look neater.